Update the description about DM collation_compatible config (#7355)
pingcap/docs: dm/task-configuration-file-full.md

online-ddl: true                # Supports automatic processing of upstream "gh-ost" and "pt".
online-ddl-scheme: "gh-ost"     # `online-ddl-scheme` has been deprecated in v2.0.6, so it is recommended to use `online-ddl`.
clean-dump-file: true           # Whether to clean up the files generated during data dump. Note that these include `metadata` files.

> collation_compatible: "loose"   # The way to sync default collation in create sql, it can be "loose" or "strict" and "loose" is default. DM will sync default collation from upstream when it is "strict", but not when it is "loose". Note that when it is "strict" and the default collation in upstream doesn't be supportted by downstream, it may occur error. 

Reviewer: ```suggestion
collation_compatible: "loose"   # The mode to sync the default collation in `CREATE` SQL statements. The supported values are "loose" (by default) or "strict". When the value is "strict", DM syncs the default collation from the upstream; when the value is "loose", DM does not sync the default collation. In "strict" mode, if the downstream does not support the default collation in the upstream, the downstream may report an error. 
```
@WizardXiao PTAL

Author: yes, that will be much clearer.

> collation_compatible: "loose"   # The mode to sync the default collation in `CREATE` SQL statements. The supported values are "loose" (by default) or "strict". When the value is "strict", DM syncs the default collation from the upstream; when the value is "loose", DM does not sync the default collation. In "strict" mode, if the downstream does not support the default collation in the upstream, the downstream may report an error. 

Reviewer 2: ```suggestion
collation_compatible: "loose"   # The mode to sync the default collation in `CREATE` SQL statements. The supported values are "loose" (by default) or "strict". When the value is "strict", DM syncs the default collation from the upstream; when the value is "loose", DM does not sync the default collation. In "strict" mode, if the downstream does not support the default collation in the upstream, the downstream might report an error. 
```

> collation_compatible: "loose"   # The mode to sync the default collation in `CREATE` SQL statements. The supported values are "loose" (by default) or "strict". When the value is "strict", DM syncs the default collation from the upstream; when the value is "loose", DM does not sync the default collation. In "strict" mode, if the downstream does not support the default collation in the upstream, the downstream might report an error. 

Reviewer 2: ```suggestion
collation_compatible: "loose"   # The mode to sync the default collation in CREATE SQL statements. The supported values are "loose" (by default) or "strict". When the value is "strict", DM syncs the default collation from the upstream; when the value is "loose", DM does not sync the default collation. In "strict" mode, if the downstream does not support the default collation in the upstream, the downstream might report an error. 
```

> collation_compatible: "loose"   # The mode to sync the default collation in `CREATE` SQL statements. The supported values are "loose" (by default) or "strict". When the value is "strict", DM syncs the default collation from the upstream; when the value is "loose", DM does not sync the default collation. In "strict" mode, if the downstream does not support the default collation in the upstream, the downstream might report an error. 

Reviewer 3: ```suggestion
collation_compatible: "loose"   # The mode to sync the default collation in `CREATE` SQL statements. The supported values are "loose" (by default) or "strict". When the value is "strict", DM appends the according collation of the upstream to SQL; when the value is "loose", DM does not modify the SQL. In "strict" mode, if the downstream does not support the default collation in the upstream, the downstream might report an error. 
```
